modyn/metadata_processor/processor_strategies/processor_strategy_type.py

```diff
@@ -13,7 +13,7 @@ class ProcessorStrategyType(Enum):
     BasicProcessorStrategy = "basic_processor_strategy"  # pylint: disable=invalid-name
 
 
-class InvalidProcessorStrategyTypeException(Exception):
+class InvalidProcessorStrategyTypeError(Exception):

```

Comment:
Why this change?

Reply:
pep8 suggests this: see https://docs.astral.sh/ruff/rules/error-suffix-on-exception-name/